concept-anchor.md

```diff
@@ -0,0 +1,36 @@
+## Title
+Concept Anchor for discussions with Management
+
+## Patlet
+Those wanting to bring InnerSource to organizations have difficulty starting a discussion with management. This can sometimes be made easier by anchoring InnerSource to more well known concepts such as Agile / DevOps / Lean.
+
+## Problem
+Often InnerSource is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting InnerSource.
+
+## Context
+An individual sees an opportunity for InnerSource to make an impact within an organization. They would like to discuss InnerSource with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of InnerSource with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build InnerSource on top of these concepts.
+
+## Forces
+* Management are reluctant to talk to you unless you can tie projects in to an existing corporate strategy.
+* Starting to explain InnerSource may be difficult unless one starts with a point decision makers are familiar with.
+* Agile/DevOps/Lean have already been heavily marketed and are commonly discussed terms in corporations, conferences, magazines etc.
+
+## Solutions
+**TBD**
```

Comment:
Do you have thoughts already how you want to anchor InnerSource in these other concepts? Even unproven ideas here could help others to understand if they have already seen this pattern in the wild.

Reply:
Writing something up now.

Reply:
Submitted some text for review. :)

Reply:
This looks great! I left one comment inline.

Feel free to mark this conversation thread as resolved.